README.md

```diff
@@ -0,0 +1,28 @@
+[![Join the chat at https://gitter.im/ethereum/remix](https://badges.gitter.im/ethereum/remix.svg)](https://gitter.im/ethereum/remix?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+# Remix Desktop
+**Remix Desktop** is an Electron version of Remix IDE.  It works on Linux, Windows, & Macs.
+
+It is not a web app - but like the name says, it is a desktop app.  To find out more about Remix IDE - please go to the Remix-project repo.
```

Comment:
under the hood we are running the same code as remix-ide, so in a sense, this is a web app...